pkgs/tools/filesystems/davfs2/default.nix

```diff
@@ -10,11 +16,21 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ neon zlib ];
 
-  patches = [ ./isdir.patch ./fix-sysconfdir.patch ];
+  patches = [
+    ./isdir.patch
+    ./fix-sysconfdir.patch
+    (substituteAll {
+      src = ./0001-umount_davfs-substitute-ps-command.patch;
+      ps = "${procps}/bin/ps";
+    })
+  ];
 
   configureFlags = [ "--sysconfdir=/etc" ];
 
-  makeFlags = ["sbindir=$(out)/sbin" "ssbindir=$(out)/sbin"];
+  makeFlags = [
+    "sbindir=$(out)/sbin"
+    "ssbindir=$(out)/sbin"
```

Comment:
is `$(out)` or `${placeholder "out"}` preferred?

Reply:
I don't know - I didn't touch that, just reformatted it. ;-)